src/components/Common/SearchByAnchor/SearchByAnchor.jsx

```diff
@@ -77,6 +92,20 @@ export default class AddTrustFromFederation extends React.Component {
                     </MessageRow>
                 );
             break;
+        case 'invalid_domain':
+            assetResults = (
+                    <MessageRow isError>
+                        <span>Invalid domain {anchorDomain}</span>
+                    </MessageRow>
+                );
+            break;
+        case 'without_currencies':
+            assetResults = (
+                    <MessageRow isError>
+                        <span>Domain {anchorDomain} is known for Stellar, but no currencies found</span>
```

Comment:
```suggestion
                        <span>Domain {anchorDomain} is known for Stellar, but no currencies found</span>
```

<span>No currencies found in the stellar.toml file</span>